Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY package-lock.json .
 COPY tsconfig.json .
 RUN npm install -g npm@9.8.1
 RUN npm install --include=dev
-RUN REACT_APP_BACKEND_URL=$REACT_APP_BACKEND_URL npm run build
+RUN npm run build
```

Comment:
I don't understand how this will work since right now, the backend URL is part of the generated assets at build time so it does need this value. Although it would be desirable to have a way to pass on a dynamic config (perhaps a JSON that is also retrieved when it loads into the browser), I don't think the code as be customized to do this.

Reply:
We can rebuild the `build` package within the container inside Kubernetes. The tradeoff of doing so is the time it will take to start the pod (we can estimate an additional 1-2 minutes to have the pod up and ready).

Reply:
as discussed, rebuild should not be necessary if we use a relative URL /api as the backend URL config and have the web frontend (nginx) upstream to the correct backend.